packages/module/src/DetailsPageHeader/DetailsPageHeader.tsx

```diff
@@ -5,18 +5,25 @@ import {
   Text,
   TextVariants,
   Label,
-  DropdownPosition,
 } from '@patternfly/react-core';
-import { isEmpty } from 'lodash';
 import React from 'react';
-import type { K8sResourceCommon } from '../../types/k8s';
-import type { BreadcrumbProp, ActionButtonProp, ActionMenuProps } from './utils';
-import { Breadcrumbs, ActionButtons, ActionMenu } from './utils';
+import {
+  ActionButtonProps,
+  ActionButtons,
+  ActionMenu,
+  ActionMenuProps,
+  BreadcrumbProps,
+  Breadcrumbs,
+} from './utils';
 import '@patternfly/react-styles/css/utilities/Spacing/spacing.css';
 
-export type PageHeading = {
-  /** Optional title for page heading */
-  title?: string;
+export interface PageHeading {
+  /** Title for page heading */
+  title: string;
+  /** Optional icon for page heading (appears to the left of the page heading's title) */
+  iconBeforeTitle?: React.ReactNode;
+  /** Optional icon for page heading (appears to the right of the page heading's title) */
+  iconAfterTitle?: React.ReactNode;
   /** Optional label for page heading */
   label?: {
```

Comment:
Should use https://github.com/patternfly/patternfly-react/blob/cb753616f434b5fc8308a19047867f16e4cceb65/packages/react-core/src/components/Label/Label.tsx#L11-L72 instead